### ISSUE TITLE
Control scheduling room with same date and time 

### ISSUE THREAD

**Issue by iDcodes** [benawad/dogehouse#928]
Hello, I just notice that I can schedule a room with the same date and time. I think it is necessary if you will control the same date and time room for one host. I hope you understand what I mean. 

![Screenshot_2021-03-14-14-00-12-339_com android chrome](https://user-images.githubusercontent.com/40931945/111059286-5e4af700-84cf-11eb-82af-77ea9d779ee7.jpg)

Also looking forward for this project! Good luck!

**Comment by ermalsh** [comment]
Maybe grouping them by time in the UI would work?

**Comment by iDcodes** [comment]
> Maybe grouping them by time in the UI would work?

That would work, but it will be another data redundancy because I still can create new schedule room on my end with the same date and time.

**Comment by mklemersson** [comment]
I agree that this might be redundant data unless, just something that might not be the case, let's say that a speaker could send its message to more than one room like an alert message but, I think that this is something that will unlikely happen.

**Comment by Rohit-Sharma14** [comment]
hello i am new to open source  but this seems like i can fix this can i do it or someone has fixed it??
